contracts/solidity/scoping/Scoping.sol

```diff
@@ -18,7 +18,6 @@ contract Scoping {
         uint x = 1;
         {
             x = 2; // this will assign to the outer variable
-            uint x;
```

Comment:
I believe this is required to showcase that what looks like a declaration of a default `uint` doesn't affect the value of `x`

Reply:
I removed it due to a warning, then I reached out to Stefan and he sent me the link to the function on Soliditylang website and you're right this line is intended to be added here. Pushed in the fix!